elisp-bytecode.texi

```diff
@@ -1029,7 +1093,10 @@ Call @code{+} with two arguments, adding the two values at the top of the stack.
 @node mult
 @unnumberedsubsubsec @code{mult} (95)
 @kindex mult
-Call @code{*} with two arguments, multiplying the two values at the top of the stack.
+Call @code{*} with two arguments, multiplying the two values at the
+top of the stack.
+
+Since emacs 19.34
```

Comment:
bytecomp.el incorrectly states "v19 only" for this opcode, but it's still emitted.